jcli/src/jcli_app/transaction/mod.rs

```diff
@@ -44,9 +44,13 @@ pub enum Transaction {
     Finalize(finalize::Finalize),
     /// Finalize the transaction
     Seal(seal::Seal),
-    /// get the Transaction ID from the given transaction
-    /// (if the transaction is edited, the returned value will change)
+    /// get the Fragment ID from the given 'sealed' transaction
+    FragmentId(common::CommonTransaction),
```

Comment:
This is where things are interesting, this will be also the `Id` that can be use to lookup the status of a transaction within the blockchain. May it be a certificate or a basic transaction.